### ISSUE TITLE
A question of details of FLASH

### ISSUE THREAD

**Issue by Smityz** [Snapchat/KeyDB#561]
I have read https://docs.keydb.dev/docs/flash about FLASH function, which is wonderful!
But I still have some problems with it.
1. How can I get the flashed data on the flash(stored in RocksDB), can I read them by KeyDB directly?
2. If my KeyDB instance restarts can it load data from RocksDB?
3.  Is any plan to support remote kvstore replace of RocksDB?

**Comment by msotheeswaran** [comment]
1. yes you read and write through KeyDB however rocksdb stores the data so that it is also on disk and can be loaded when KeyDB is restarted
2. yes
3. can you elaborate what you mean by this, KeyDB is still a kvstore rocksdb just persists the data in KeyDB on disk

**Comment by Smityz** [comment]
iStorage is an interface, and I want to know if the interface can be replaced by a remote KVstore(DynamoDB/CosmosDB/HBase). This way, we may be able to get higher write throughput and better data persistence.

**Comment by msotheeswaran** [comment]
Of course! You can implement the IStorage interface using any other storage medium and KeyDB will persist to it.

**Comment by Smityz** [comment]
Thanks for your answer!
